cmd/dtab.go

```diff
@@ -124,7 +138,7 @@ var (
 				if err != nil {
 					return err
 				}
-				_, err = ctl.Update(name, dtabstr, namer.Version(dtabUpdateVersion))
+				_, err = ctl.Update(name, dtabstr, dtabJson, namer.Version(dtabUpdateVersion))
```

Comment:
Same goes for update as well.  Maybe I don't have the right version of namerd?  I just re-pulled the nightly docker image.